gce_workload_cert_refresh/main.go

```diff
@@ -262,6 +275,20 @@ func refreshCreds() error {
 		return fmt.Errorf("Error writing config_status: %v", err)
 	}
 
+	// Handles the edge case where the config values provided at VM creation may be invalid. This ensures
```

Comment:
i think at first run is clearer than at VM creation, because this can be enabled post-boot, right?